test/templates/config/UseSecondary.py

```diff
@@ -0,0 +1,29 @@
+from WMCore.Configuration import Configuration
+config = Configuration()
+
+#General Section
+config.section_("General")
+config.General.requestName = 'CHANGE'
+config.General.workArea = 'CHANGE'
+config.General.transferOutputs = True
+config.General.transferLogs = True
+config.General.instance = 'preprod'
+
+#Job Type Section
+config.section_("JobType")
+config.JobType.pluginName = 'Analysis'
+config.JobType.psetName = 'psets/pset_parent_analysis.py'
```

Comment:
Is this the same as `psets/pset_use_parent.py` ? this file which is in config does not exist.